crucible.mjs

```diff
@@ -269,6 +271,21 @@ Hooks.once("init", async function() {
   CONFIG.debug.talentTree = false;
   CONFIG.debug.flanking = false;
   if ( crucible.developmentMode ) registerDevelopmentHooks();
+
+  // Add Hit Box layer
+  CONFIG.Canvas.layers.crucibleHitBoxes = {
```

Comment:
What's your rationale for this being a new canvas layer rather than part of the existing grid layer?

Reply:
We don't override/monkeypatch anything. This is all using the Foundry API, putting the new layer in CONFIG, and having our class with 100% control on it. This is also very readable and easily maintained.